### ISSUE TITLE
About the pre-train strategy in the code

### ISSUE THREAD

**Issue by Loneseven** [OpenGVLab/InternVL#491]
At the moment the internvl_chat/internvl/train/internvl_chat_finetune.py and internvl_chat/internvl/train/internvl_chat_pretrain.py are almost the same. I am wondering if the ITM loss and Contrastive loss would be used. Am I miss something? Is generative loss alone enough?

**Comment by czczup** [comment]
Hello, when training the chat model, we don't use ITM loss or contrastive loss; using generative loss alone is sufficient.

**Comment by Loneseven** [comment]
Thanks for the reply. In the original InternVL paper we see: stage 1: contrastive pre-training. So It's not necessary now for the newer versions 1.5 and 2? I mean the released ckpts.

**Comment by czczup** [comment]
In InternVL 1.0, we trained the first version of [InternViT-6B-224px](https://huggingface.co/OpenGVLab/InternViT-6B-224px) using contrastive learning. After that, we connected this model with an LLM for incremental pretraining, gradually developing [InternViT-6B-448px-V1-0](https://huggingface.co/OpenGVLab/InternViT-6B-448px-V1-0), [InternViT-6B-448px-V1-2](https://huggingface.co/OpenGVLab/InternViT-6B-448px-V1-2), and [InternViT-6B-448px-V1-5](https://huggingface.co/OpenGVLab/InternViT-6B-448px-V1-5). Therefore, contrastive learning is no longer necessary for training these later models.

**Comment by czczup** [comment]
You can look at the model tree of our vision encoders:

<img width="633" alt="image" src="https://github.com/user-attachments/assets/eb1669ad-a258-48a0-97cf-bed323870a01">
